papeleta-pdf-extractor/src/config/2020/base-config.ts

```diff
@@ -0,0 +1,223 @@
+export interface BaseRow {
+  IMAGE_NAME_PREFIX?: string;
+  MAIN_TOP?: number;
+  MAIN_HEIGHT?: number;
+  hasLogos?: boolean;
+}
+export interface Row extends BaseRow {
+  N_LOGO?: number;
+  LOGO_TOP?: number;
+  LOGO_HEIGHT?: number;
+  LOGO_LEFT_DELTA?: number;
+  LOGO_RIGHT_DELTA?: number;
+}
+interface DefaultValues {
+  [index: string]: BaseRow | Row;
+}
+const defaultRowValues: DefaultValues = {
```

Comment:
The default values are for a legislative ballot and not general purpose. We might want to have separate configs or base classes (Legislative, Municipal, etc) instead.

How would a config look like using this method?

Reply:
@Jpadilla1 This is just an initial stab at it. 

If this approach would be something that we can use I'd make base configs for municipal and legislative ballots. Making a config module that exposes a single API for configs will be useful IMO.

If we want to continue with this I'll structure it better and make this useable.